br/pkg/restore/client.go

```diff
@@ -64,12 +67,17 @@ func NewRestoreClient(
 	keepaliveConf keepalive.ClientParameters,
 	isRawKv bool,
 ) (*Client, error) {
+	apiVerion, err := backup.GetTiKVApiVersion(context.Background(), pdClient, tlsConf)
```

Comment:
It's a little strange to use a method from `backup` in restore procedure. It would be better to move `GetTiKVApiVersion` as a common method.

Reply:
moved to common `conn` package, thanks for the good suggestion.